master/reference/cni-plugin/configuration.md

```diff
@@ -285,6 +285,12 @@ If provided, these IP pools will override any IP pools specified in the CNI conf
 > match a configured [IPPool]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool) resource.
 {: .alert .alert-info}
 
+> **Note2**: CNI Plugin supports specifying per namespace annotation.
```

Comment:
Thanks for your contribution @rikatz . We do not have the convention of numbering our notes. If you can remove the "2", that will be better.

Also, I am not sure why POD is in all caps. If you are referring to a Kubernetes pod, then it should be all lowercase.

You are missing a few articles and you have capitalized some words that are not proper nouns. Therefore, you could:
- s/CNI plugin/The {{site.prodname}} CNI plugin
- s/specifying per namespace annotation/specifying an annotation per namespace
- s/both Namespace and POD/both the namespace and the pod
- s/only Namespace have annotation/only the namespace has the annotation

Also, it is unclear what "this" refers to. May I suggest clarifying this as follows:
- s/this/the annotation of the namespace

These are all minor points, but if you could take the time to make the changes, we would appreciate it. Thanks for the PR!

Reply:
No problems!

I've made the recommended corrections